### PR TITLE
add demonstration of log config of "nginx_config" in docs.

### PR DESCRIPTION
**Pull request by kerneltravel**
### What this PR does / why we need it:
add demonstration in log format of nginx_config customization for somebody who need an example of this subject.
and fixed the original incorrect indents.
[](https://github.com/apache/apisix/issues/6072)

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by leslie-tsang** [comment]
Hello there, Can you add a test like `fix #6072` in PR desc, it help reviewer to sync status between issues and PR.

File: docs/en/latest/customize-nginx-configuration.md [hunk 1]
```diff
@@ -25,37 +25,78 @@ The Nginx configuration used by APISIX is generated via the template file `apisi
...
 You can take a look at the generated Nginx configuration in `conf/nginx.conf` after running `./bin/apisix start`.
 
-If you want to customize the Nginx configuration, please read through the `nginx_config` in `conf/config-default.yaml`. You can override the default value in the `conf/config.yaml`. For instance, you can inject some snippets in the `conf/nginx.conf` via configuring the `xxx_snippet` entries:
+If you want to customize the Nginx configuration, please read through the `nginx_config` in `conf/config-default.yaml`. You can override the default value in the `conf/config.yaml`. For instance, you can custum access_log_format via configuring the `http` entries; you can inject some snippets in the `conf/nginx.conf` via configuring the `xxx_snippet` entries:
 
 ```yaml
 ...
 # put this in config.yaml:
 nginx_config:
-    main_configuration_snippet: |
```
Comment (spacewander) [inline docs/en/latest/customize-nginx-configuration.md:34]: I don't think the original indent (4 spaces) is wrong.

**Review by spacewander — CHANGES_REQUESTED** [review]

**Comment by spacewander** [comment]
There is no need to put everything into the documentation. We just document what's critical, otherwise the doc is too long to read.

**Comment by spacewander** [comment]
@kerneltravel 
Anyway, I think the suggestion below is worthwhile.

> Pay attention to the indent of `nginx_config` and sub indent of the sub entries, the incorrect indent may cause `./bin/apisix start` to fail to generate Nginx configuration in `conf/nginx.conf`.

Would you mind just submitting this phrase in another PR? Thanks!

**Pull request closed without merging**